lib/PSR/HTTP/WebSocket/Events.hs

```diff
@@ -0,0 +1,25 @@
+module PSR.HTTP.WebSocket.Events where
```

Comment:
Moved from `PSR.Websocket.Server`. Also changed to `WebSocket` to match the capitalisation used by the websockets package.